docs/UserGuide.md

```diff
@@ -13,7 +13,131 @@
 
 ## Features 
 
-{Give detailed description of each feature}
+
+
+
+
+
+
+
+## 3. Academic
+Have you ever had to dig through tons of emails to look for the contact of a particular TA 
+or sweat over your calculator when trying to estimate your CAPS this semester? 
+Academic tracker provides a convenient experience where you can store 
+all the information you need in one place! To access the academic tracker, 
+follow the instructions below.
+
+###3.1 Entering Academic Mode : Entering Academic Mode : 
+`cd 4` / `cd academic`
+
+This command allows you to enter academic tracker mode from the main menu or from the other modes.
+After entering this mode, you can then access all the features that the academic tracker offers!
+
+Instruction: 
+
+1. `cd 4` / `academic`
+
+Expected output:
+![Academic_3_1](Images/AcademicUG/Academic_3_1.png)
+
+### 3.2 Adding a contact : 
+`add contact c/CONTACT DETAILS m/MOBILE NUMBER  e/EMAIL`
+
+In academic mode, you can add a contact to the current list of contacts, following the instructions below. 
+
+Instruction: 
+1. `add contact c/CONTACT DETAILS  m/MOBILE NUMBER  e/EMAIL`
+
+Example of usage: 
+
+`add contact c/Prof Lim  m/81234567  e/E7654321@u.nus.edu` 
+will add a contact with the name Prof Lim, mobile number 81234567, and email E7654321@u.nus.edu.
+
+Expected output:  
+![Academic_3_2](Images/AcademicUG/Academic_3_2.png)
+
+*Note that contacts must be fully numerical and emails must contain an @.
```

Comment:
Is this supposed to be a bullet point? If so, an extra space is needed after *

Perhaps you can consider putting it in under **Useful information:**